src/stream.cpp

```diff
@@ -151,7 +151,7 @@ inline UtfIntroCharType IntroCharTypeOf(std::istream::int_type ch) {
 
 inline char Utf8Adjust(unsigned long ch, unsigned char lead_bits,
                        unsigned char rshift) {
-  const unsigned char header =
+  const auto header =
```

Comment:
Leave this one as-is; I don’t think auto helps here.

Reply:
This was automatically done. The idea is to avoid duplicating the type name since there's a static_cast.

Reply:
I know, but I don’t agree with the philosophy. I prefer auto only when (1) the type is obvious and (2) the type name is unwieldy to write. This matches (1) but not (2).

Reply:
OK. Added nolint.

Reply:
Could you keep off the NOLINT? I don’t think that adds anything either, since there are so many different linters we could use.

Reply:
OK.